website/pages/index.vue

```diff
@@ -8,9 +8,9 @@ import waveSvg from "~/public/waveNounProject.svg";
 //       but this should ideally be fetched from somewhere, or provided in a yml etc.
 //       Could use any other identifier instead of trsId that seems fit
 const POPULAR_WORKFLOW_TRS_IDS = [
+    "#workflow/github.com/iwc-workflows/chipseq-pe/main",
+    "#workflow/github.com/iwc-workflows/rnaseq-pe/main",
```

Comment:
If that defines the order I would start with `rnaseq-pe`
```suggestion
    "#workflow/github.com/iwc-workflows/rnaseq-pe/main",
    "#workflow/github.com/iwc-workflows/chipseq-pe/main",
```